src/index.html

```diff
@@ -1,13 +1,349 @@
+                  </a>
+
+                  <a href="#" class="
+                  category__link
+                  category__link--wide">
+                    <img
+                    class="category__photo"
+                    src="images/images-category/image-6.png"
+                    alt="Audio sunglasses"
+                    >
+                  </a>
+                </div>
+
+                <h3 class="category__title">
+                  Audio sunglasses
+                </h3>
+                <div class="button-container">
+                  <div class="category__button-field">
+                    <a class="category__button" href="#">
+                      View all
+                    </a>
+                  </div>
+                </div>
+              </article>
+            </div>
+          </section>
+
+          <section class="how-to-buy" id="how-to-buy">
+            <h2 class="
+            how-to-buy__title
+            section-title
+            section-title--align--left
+            ">
+            Why buy direct from Bose
+            </h2>
+
+            <div class="how-to-buy__content">
+              <ul class="how-to-buy__benefits">
+                <li>Free 2-day shipping and returns</li>
+                <li>90-day risk-free trial</li>
+                <li>World class customer service</li>
+                <li>My Bose account management</li>
+              </ul>
+
+              <p class="how-to-buy__description">
+                A great product is more than what’s in the box. It’s a promise of premium performance, world-class support, and everything you expect from a trusted brand. It’s just one of many reasons why you’ll shop with confidence on Bose.com.
+              </p>
+            </div>
+          </section>
+          <section class="contacts" id="contacts">
+            <h2 class="section-title">Contact us</h2>
+
+            <div class="contact__field">
+              <div class="contact">
+                <form  onsubmit="this.reset(); return false;" class="forma">
+                  <div>
+                    <input
+                      type="text"
+                      name="name"
+                      class="form__field"
+                      autocomplete="off"
+                      placeholder="Name"
+                      required
+                    >
+                  </div>
+                  <div>
+                    <input
+                      type="email"
+                      name="email"
+                      class="form__field"
+                      required
+                      placeholder="Email"
+                    >
+                  </div>
+                  <div>
+                    <textarea
+                      class="form__field
+                      form__field-textarea"
+                      name="Message"
+                      placeholder="Message"
+                      required
+                    >
+                    </textarea>
```

Comment:
```suggestion
                   ></textarea>
```
That causes the following issue (initially typed spaces, no placeholder):
![image](https://github.com/mate-academy/layout_landing-page/assets/86923089/6a694e02-9e55-45ca-b743-ac446cb1b9a6)